spacy/cli/find_threshold.py

```diff
@@ -183,9 +191,23 @@ def filter_config(config: Dict[str, Any], keys: List[str]) -> Dict[str, Any]:
         )
 
     best_threshold = max(scores.keys(), key=(lambda key: scores[key]))
-    if not silent:
-        print(
-            f"\nBest threshold: {round(best_threshold, ndigits=4)} with value of {scores[best_threshold]}."
+
+    # If all scores are identical, emit warning.
+    if all([score == scores[thresholds[0]] for score in scores.values()]):
```

Comment:
Very minor, but I think you could just check that there's only one unique value in `scores.values()` without referring to `thresholds`? (I'm trying to imagine cases where you have weird settings and `len(thresholds) == 0`.)